larq_compute_engine/mlir/python/converter_test.py

```diff
@@ -1,25 +1,26 @@
 import sys
-import pytest
+import unittest
```

Comment:
`pytest` will automatically import `larq_compute_engine/__init__.py` which would break our mocking setup here. This switches the test to pure Python `unittest`.